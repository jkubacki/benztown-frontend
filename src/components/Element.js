import React from 'react';

class Element extends React.Component {
  render() {
    const { element } = this.props;
    return (
      <>
        {element.id}
      </>
    )
  }
}

export default Element;
