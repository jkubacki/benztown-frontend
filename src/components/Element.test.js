import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Element } from './Element';

const props = { element: { attributes: { name: 'Element name', tag_list: ['tag1', 'tag2', 'tag3'] } } };

it('renders without crashing', () => {
  const component = shallow(<Element {...props} isLoggedIn />);
  expect(toJson(component)).toMatchSnapshot();
});
