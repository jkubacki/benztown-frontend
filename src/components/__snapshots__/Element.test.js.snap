// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<tr>
  <td>
    Element name
  </td>
  <td>
    <t
      autoPlay={false}
      className=""
      controls={true}
      controlsList=""
      crossOrigin={null}
      id=""
      listenInterval={10000}
      loop={false}
      muted={false}
      onAbort={[Function]}
      onCanPlay={[Function]}
      onCanPlayThrough={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onListen={[Function]}
      onLoadedMetadata={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onSeeked={[Function]}
      onVolumeChanged={[Function]}
      preload="metadata"
      src={null}
      style={Object {}}
      title=""
      volume={1}
    />
  </td>
  <td>
    tag1, tag2, tag3
  </td>
</tr>
`;
